src/main/cpp/faultinj/README.md

```diff
@@ -0,0 +1,98 @@
+# Fault Injection Tool for CUDA Runtime and CUDA Driver API
+
+The goal of this tool is to increase testability of the failure handling logic
+in CUDA applications.
+
+It's especially important when CUDA is embedded in a higher-level fault-tolerant framework
```

Comment:
Super pedantic nitpick: I would avoid this sort of contraction in documentation, if possible:
```suggestion
It is especially important when CUDA is embedded in a higher-level fault-tolerant framework
```